src/main/java/uk/gov/pay/ledger/transaction/state/PaymentState.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.pay.ledger.transaction.state;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Arrays.stream;
+import static java.util.List.of;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.CAPTURABLE;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.CREATED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.ERROR_GATEWAY;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.FAILED_CANCELLED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.FAILED_EXPIRED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.FAILED_REJECTED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.STARTED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.SUBMITTED;
+
+public enum PaymentState {
+
+    IN_PROGRESS(of(CREATED, STARTED, SUBMITTED, CAPTURABLE), "In progress"),
+    SUCCESS(of(TransactionState.SUCCESS), "Success"),
```

Comment:
nitpicking but could this be a static import like the rest?

Reply:
Has to be qualified as it conflicts with the SUCCESS enum of PaymentState